src/window.py

```diff
@@ -525,7 +525,7 @@ def _find_icon(self, path):
         if utils.is_file_dir(path):
             return "folder-symbolic"
```

Comment:
I mean using a GIcon would require reworking this, but it's not hard to return a GThemedIcon here.